File: programming files/datetime_convert.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        // Set string to data and time
        String dateStr = "2022-03-17 10:45:30";
        //create DateTimeFormatter object
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
        // create LocalDate time object use formatter. 
        LocalDateTime dateObj = LocalDateTime.parse(dateStr, formatter);
        // formate dateObj
        String formattedDate = dateObj.format(DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss"));
        // output date
        System.out.println(formattedDate);
    }
}

File: programming files/datetime_convert.py
# import library
from datetime import datetime

date_str = "2022-03-17 10:45:30"  # date time string

# create datetime object based on string
date_obj = datetime.strptime(date_str, '%Y-%m-%d %H:%M:%S')
# reformat datetime object
formatted_date = datetime.strftime('%m/%d/%Y %H:%M:%S')

# print the reformatted date and time
print(formated_date)
